src/v1/database_admin_client.js

```diff
@@ -295,7 +295,7 @@ class DatabaseAdminClient {
    *
    * @example
    *
-   * const {Spanner} = require('@google-cloud/spanner');
+   * const spanner = require('@google-cloud/spanner');
```

Comment:
@crwilcox @alexander-fenster eeeek.  We updated this particular module to require the `const {Spanner} = require('@google-cloud/spanner');` pattern. 

It's likely this code isn't right anymore :(  Ideas on how we could approach this?

Reply:
In the immediate, we can add a replacement pattern to mutate these in synth.py. Long term we probably want the generator to output these using the newer style.

Reply:
In fact, this is fine the way it is so leaving it be.